Small clarifications on `how-to-get-help` (#873)
opensafely/documentation: docs/how-to-get-help.md

(Your first port of call should be to ask your question in the [Q&A forum](#qa-forum).)

You may wish to direct your question to the **tech support team**.

> To do so, include the string `tech-support` (that's with a dash and no spaces) in your question

Reviewer: I'd suggest not including this. It doesn't matter that much.

> To do so, include the string `tech-support` (that's with a dash and no spaces) in your question

Reviewer 2: 😆 this string literal, by which I mean this string literal

Author: I literally mean literally this literal string

and a member of this team will aim to respond within half a working day.

> The support team membership changes regularly, so we ask that you don't send direct messages (DMs) to individual team members.

Reviewer 2: ```suggestion
The tech support team membership changes regularly, so we ask that you don't send direct messages (DMs) to individual team members.
```
nit

> The support team membership changes regularly, so we ask that you don't send direct messages (DMs) to individual team members.

Reviewer: Not sure. I don't think we should be speaking for other groups, only for the Tech Group. How about:

```suggestion
We ask that you don't send direct messages (DMs) to members of the Tech Group.
```
